modules/build-raptor/src/endpoint-requester.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios'
+
+import { GithubResponseSchema } from './build-raptor-api'
+import { getEnv } from './build-raptor-cli'
+
+export async function getPRForCommit(commitHash: string): Promise<number | undefined> {
+  if (!commitHash || !commitHash.match(/^[a-f0-9]{40}$/)) {
+    throw new Error('Invalid commit hash.')
+  }
+
+  const repoOwner = getEnv('GITHUB_REPOSITORY_OWNER')
+  const repoName = getEnv('GITHUB_REPOSITORY')
```

Comment:
let's pass these two values as parameters (i.e., the caller will call getEnv() and pass the values it gets from it as parameters)

Reply:
done